distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc

```diff
@@ -173,14 +173,14 @@ Admin Console.
 |List of imagery providers to use. Valid types are: +
  -OSM (OpenStreetMap), +
  -AGM (ArcGisMap), +
- -BM (BingMap),+
+ -BM (BingMap), +
  -WMS (WebMapService), +
  -WMT (WebMapTile), +
  -TMS (TileMapService), +
- -GE (GoogleEarth).
- -
- -Example: TYPE={url=
- -{"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}}
+ -GE (GoogleEarth), +
+ Examples: +
+ -{"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5} +
+ -{"type" "WMT" "url" "http://example.com" "layer" "layer1" "style" "default" "tileMatrixSetID" "tileSetId"}
```

Comment:
I also agree that there should be additional examples in the documentation.  These JSON strings are designed to work with both maps ( the examples provided are supported by both APIs).